ambari-server/src/main/resources/stack-hooks/before-START/scripts/params.py

```diff
@@ -254,10 +254,10 @@
 default_fs = module_configs.get_property_value(module_name, 'core-site', 'fs.defaultFS')
 
 #host info
-all_hosts = execution_command._execution_command.__get_value("clusterHostInfo/all_hosts", [])
-all_racks = execution_command._execution_command.__get_value("clusterHostInfo/all_racks", [])
-all_ipv4_ips = execution_command._execution_command.__get_value("clusterHostInfo/all_ipv4_ips", [])
-slave_hosts = execution_command._execution_command.__get_value("clusterHostInfo/datanode_hosts", [])
+all_hosts = execution_command.get_value("clusterHostInfo/all_hosts", [])
+all_racks = execution_command.get_value("clusterHostInfo/all_racks", [])
+all_ipv4_ips = execution_command.get_value("clusterHostInfo/all_ipv4_ips", [])
+slave_hosts = execution_command.get_value("clusterHostInfo/datanode_hosts", [])
```

Comment:
Why is this variable called slave_hosts? Doesn't make sense.

Reply:
this is original code:
slave_hosts = default("/clusterHostInfo/datanode_hosts", [])